packages/grafbase-sdk/README.md

```diff
@@ -2,6 +2,14 @@
 
 A TypeScript library to generate a Grafbase configuration. It replaces the `schema.graphql` file with `grafbase.config.ts`, which should be placed into the project's `grafbase` directory.
 
+## Get Started
+
+```bash
+npx grafbase init --config-format typescript
```

Comment:
We should say this is for a new project, to add it to the existing one, `(npm|npx yarn|pnpm) install @grafbase/sdk --save-dev`.